doc/Settings.md

```diff
@@ -97,6 +97,15 @@ The 'skipDependencies' behavior affects whether dependencies are installed for a
     "installBehavior": {
         "skipDependencies": true
     },
+```
+
+### Extract Archive With Tar Executable
+The 'extractArchiveWithTar' behavior affects whether installer archives are extracted using the tar executable ('tar.exe'). Defaults to 'false' if value is not set or is invalid.
+
+```json
+    "installBehavior": {
+        "extractArchiveWithTar": true
```

Comment:
I was expecting a string setting like below for extensibility

"archiveExtraction": "tar"  allowed values are (tar, shellApi)

Reply:
changed to archiveExtractionMethod that takes in string values.